4/typedefs.js

```diff
@@ -0,0 +1,214 @@
+const uuid = require('uuid/v1');
+
+const typeDefs = `
+
+  enum State {
+    DONE
+    UNDONE
+    IN_PROGRESS
+  }
+
+  enum Order {
+    DESC
+    ASC
+  }
+
+  type Todo {
+    id: ID!
+    message: String!
+    state: State
+    assignee: Assignee @relation(name: "ASSIGNED_TO", direction: "IN")
+  }
+
+  type Assignee {
+    id: ID!
+    name: String!
+  }
+
+  type Query {
+    todos: [Todo]
+    todo(id: ID!): Todo
+    todosOrderedByState(sortOrder: Order!): [Todo]
+    skipAndLimitTodos(skip: Int, limit: Int): [Todo]
+  }
+
+  type Mutation {
+    createAssignee(id: ID, name: String!): Assignee
+    createTodo(id: ID, message: String!, state: State): Todo
+    createRelationship(id: ID!, name: String!): Todo
+    deleteTodo(id: ID!): Todo
+    deleteAllTodos: Boolean
+    deleteAllAssignees: Boolean
+    setTodoState(id: ID!, state: State!): Todo
+  }
+
+`;
+
+const resolvers = {
+  Query: {
+    todos: async (object, args, context) => {
+      const session = context.driver.session();
+      
+      try {
+        const cypherQuery = 'MATCH (t:Todo) RETURN t';
```

Comment:
```suggestion
        const cypherQuery = 'MATCH (t:Todo) RETURN t {.*}';
```
Then you can skip the `.properties` on line 56.